tests/jballerina-unit-test/src/test/resources/test-src/statements/vardeclr/AccessProjectNegative/modules/module1/module1.bal

```diff
@@ -0,0 +1,19 @@
+//  Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Please recheck the naming convention for projects too. IIRC, we use lower case letters with hyphens or underscores.